.github/workflows/sync-master-develop.yml

```
@@ -0,0 +1,16 @@
+name: Sync master/develop branches
+on:
+  pull_request:
+    types: [closed]
+    branches: master
+jobs:
+  create_sync_pull_request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dequelabs/action-sync-branches@v1.0.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-title: "chore: merge master into develop"
+          pr-reviewers: straker,WilcoFiers
```

May as well add me here too. I'm happy to look at these PRs.

```suggestion
          pr-reviewers: straker,WilcoFiers,stephenmathieson
```